demo.ipynb

```diff
@@ -98,40 +89,36 @@
    "execution_count": null,
    "id": "26208251-08a0-457f-93f0-eda4c4a7cc22",
    "metadata": {
+    "scrolled": true,
     "tags": []
    },
    "outputs": [],
    "source": [
-    "unset TASKS\n",
-    "declare -A TASKS\n",
     "export HOME=/tmp\n",
     "for KEY in \"${!TES_INSTANCES[@]}\"; do\n",
     "    TES=\"${TES_INSTANCES[$KEY]}\"\n",
     "    echo \"Submitting task to $KEY ($TES)...\"\n",
-    "    TASK_ID=$( \\\n",
-    "        snakemake --tes \"${TES%/}\" --snakefile Snakefile_s3 --jobs 1 --cores 1 -F --rerun-incomplete --envvars HOME  | \\\n",
-    "        jq \".id\" - | \\\n",
-    "        tr -d '\"'\n",
-    "    )\n",
-    "    if [[ $TASK_ID == \"null\" ]]; then\n",
-    "        echo \"FAILED\"\n",
-    "    else\n",
-    "        echo \"Task ID: $TASK_ID\"\n",
-    "        TASKS[[ $TASK_ID ]]=\"$TES\"\n",
-    "    fi\n",
+    "    snakemake \\\n",
+    "        --directory wf-hashsplitter \\\n",
+    "        --jobs 1 \\\n",
```

Comment:
add  `--snakefile wf-hashsplitter/Snakefile \`

Reply:
It's not necessary if the file is called `Snakefile` and inside `--directory`. But I agree, why not be explicit. I'll add it together with the credentials